src/neo/Network/P2P/MessageCommand.cs

```diff
@@ -33,6 +33,7 @@ public enum MessageCommand : byte
         GetData = 0x28,
         [ReflectionCache(typeof(GetBlockByIndexPayload))]
         GetBlockByIndex = 0x29,
+        [ReflectionCache(typeof(InvPayload))]
```

Comment:
Something miss here?

Reply:
In #1397 this was deleted. Should we restore this?
![image](https://user-images.githubusercontent.com/43407364/106081693-4f44fb00-6154-11eb-9649-41628842f8e2.png)

Reply:
Yes. It should use `InvPayload`.

Reply:
This asociate NotFound with InvPayload, in https://github.com/neo-project/neo/pull/1397/files#diff-6d6b523c81ceea7eab9d910f38439af06d6912a0fb6e43680056db6f1abc9e08R36 it was not asociated.

Reply:
This deletion should have been a typo